scripts/mkcloud

```diff
@@ -540,24 +540,84 @@ function setupadmin()
     fi
 }
 
+function shutdown_node() {
+    virsh shutdown $1
+}
```

Comment:
Well, it's true that it's a very short function :smirk: I figured though that if we ever want to change the virtulization approach we just have to change the function.